sage/group_prover.sage

```diff
@@ -100,14 +100,19 @@ class fastfrac:
     """Multiply something else with a fraction."""
     return self.__mul__(other)
 
-  def __div__(self,other):
+  def __truediv__(self,other):
     """Divide two fractions."""
     if parent(other) == ZZ:
       return fastfrac(self.R,self.top,self.bot * other)
     if other.__class__ == fastfrac:
       return fastfrac(self.R,self.top * other.bot,self.bot * other.top)
     return NotImplemented
 
+  # Compatibility wrapper for Sage versions based on Python 2
+  def __div__(self,other):
+     """Divide two fractions."""
+     return __truediv__(self,other)
```

Comment:
`NameError: global name '__truediv__' is not defined`. This should be `return self.__truediv__(other)`

Reply:
Ok, I assumed I'm clever enough to write three lines of Python code without testing. But apparently I was wrong. (Sorry, it's a lot of work for me to install an older sage version...) Should work now.

Reply:
Fwiw, `nix-shell -p sage` would work but the tests take forever. Therefore you can use `nix-shell '-p sage.overrideAttrs (oldAttrs: rec { buildInputs = [ makeWrapper ]; })'` to disable the tests.